server/routerlicious/packages/routerlicious/src/scribe/index.ts

```diff
@@ -53,6 +53,8 @@ export async function scribeCreate(
 	const getDeltasViaAlfred = config.get("scribe:getDeltasViaAlfred") as boolean;
 	const transientTenants = config.get("shared:transientTenants") as string[];
 	const localCheckpointEnabled = config.get("checkpoints:localCheckpointEnabled") as boolean;
+	const restartOnCheckpointFailure =
```

Comment:
Nit: When we introduce a new config, we usually add it to config.json and the configmap even if we want it to fallback to the defaults in oss, so that we can keep an inventory of the configs